.github/workflows/release.yml

```
@@ -141,9 +192,94 @@ jobs:
+          {
+            echo "message<<EOF"
+            echo "### Devcontainer Image"
+            echo ""
+            echo "\`$FULL_IMAGE\`"
+            echo "EOF"
+          } >> $GITHUB_OUTPUT
+
+      - name: Generate PyPI Link
+        id: pypi_link
+        if: needs.pypi.outputs.should_publish == 'true' && needs.pypi.result == 'success'
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:20:6: Double quote to prevent globbing and word splitting

(shellcheck)